src/api/app/controllers/webui/driver_update_controller.rb

```diff
@@ -107,7 +107,7 @@ def save
   # TODO: select architecture of binary packages
   def binaries
     required_parameters :repository
-    @binaries = @package.build_result(params[:repository], ['binarylist', 'status']).each('//binary').map{|binary| binary['filename']}
+    @binaries = @package.build_result_old(params[:repository], ['binarylist', 'status']).each('//binary').map{|binary| binary['filename']}
```

Comment:
Is it possible for this to use the new `@package.buildresults` method? So we can get rid of this `build_result_old` method? Or maybe do that in a card on the next sprint?